airflow/providers/amazon/aws/operators/s3_delete_objects.py

```diff
@@ -56,22 +59,34 @@ class S3DeleteObjectsOperator(BaseOperator):
     :type verify: bool or str
     """
 
-    template_fields = ('keys', 'bucket')
+    template_fields = ('keys', 'bucket', 'prefix')
 
     @apply_defaults
     def __init__(
             self,
             bucket,
-            keys,
+            keys=None,
+            prefix=None,
             aws_conn_id='aws_default',
             verify=None,
             *args, **kwargs):
+
+        if not keys and not prefix:
+            raise ValueError("Either keys or prefix should be set. Both are None")
```

Comment:
```suggestion
        if bool(keys) != bool(prefix):
            raise ValueError("Either keys or prefix should be set.")
```

Reply:
Thanks for your review.

I missed the possibility of an empty array in the keys. When an empty array is passed to the keys, S3DeleteObjectsOperator does nothing without raising an exception. I'm going to compare the condition to None in order to not change this behavior.

```python
if keys is None and prefix is None:
```

I'm also going to add a simple test to confirm this behavior.

Reply:
> bool(keys) ^ bool(prefix)

..is basically an `XOR` which means it will raise an error if either both values are None or both values are set.

Reply:
I now understand the need to consider cases where both values are included. Thanks. 👍 
Fixed it at 96806fa.